DevCycle.SDK.Server.Local.MSTests/EnvironmentConfigManagerTest.cs

```diff
@@ -47,8 +47,8 @@ private Tuple<EnvironmentConfigManager, MockHttpMessageHandler, MockedRequest> g
             var cfgManager = new EnvironmentConfigManager(sdkKey, new DevCycleLocalOptions(),
                 loggerFactory, new LocalBucketing(), restClientOptions: new DevCycleRestClientOptions()
                     { ConfigureMessageHandler = _ => mockHttp },
-                initializedHandler: (isError && !isRetryableError)
-                    ? DidNotInitializeSubscriber
+                initializedHandler: isError
+                    ? isRetryableError ? DidNotInitializeRetryableSubscriber : DidNotInitializeSubscriber
```

Comment:
this would probably be more understandable with some brackets / formatting:
```suggestion
                    ? (isRetryableError ? DidNotInitializeRetryableSubscriber : DidNotInitializeSubscriber)
```

Reply:
I tried to make it more clear via the newlines.